forum/qa-plugin/user-changeable-theme/themes/dark/src/main.scss

```diff
@@ -1,26 +1,27 @@
-@import 'var/var';
```

Comment:
Dlaczego zamiana `'` na `"`?

Reply:
Autoformatowanie w mojej globalnej konfiguracji prettier, mogę to pozmieniać jak chcesz